src/cpu.cpp

```diff
@@ -508,21 +508,28 @@ static CPUPowerData_k10temp* init_cpu_power_data_k10temp(const std::string path)
 
     std::string coreVoltageInput, coreCurrentInput;
     std::string socVoltageInput, socCurrentInput;
+    std::string socPowerInput, corePowerInput;
 
     if(!find_input(path, "in", coreVoltageInput, "Vcore")) return nullptr;
     if(!find_input(path, "curr", coreCurrentInput, "Icore")) return nullptr;
     if(!find_input(path, "in", socVoltageInput, "Vsoc")) return nullptr;
     if(!find_input(path, "curr", socCurrentInput, "Isoc")) return nullptr;
+    if(!find_input(path, "power", corePowerInput, "Pcore")) return nullptr;
```

Comment:
With your k10 patch you only need socPower and corePower, right? it's kind of redundant to include the others and print them to debug since they won't be used